src/Provider/AbstractProvider.php

```diff
@@ -399,37 +402,21 @@ protected function parseResponse($response)
     abstract protected function checkResponse(array $response);
 
     /**
-     * Prepare the access token response for the grant. Custom mapping of
-     * expirations, etc should be done here.
+     * Prepare the access token response for the grant.
+     *
+     * Custom mapping of expirations, etc should be done here. Always call the
+     * parent method when overloading this method!
      *
      * @param  array $result
      * @return array
      */
     protected function prepareAccessTokenResult(array $result)
     {
-        $this->setResultUid($result);
+        $result['uid'] = $result[static::ACCESS_TOKEN_UID];
         return $result;
     }
 
     /**
-     * Sets any result keys we've received matching our provider-defined uidKey to the key "uid".
-     *
-     * @param array $result
-     */
-    protected function setResultUid(array &$result)
```

Comment:
I didn't see a reason to have this entirely separate method when we have `prepareAccessTokenResult` instead. I think it was vestigial from methods added <1.0.